geomesa-tutorials-accumulo/geomesa-tutorials-accumulo-authorizations/src/main/resources/META-INF/services/org.locationtech.geomesa.security.AuthorizationsProvider

```diff
@@ -0,0 +1 @@
+org.geomesa.example.accumulo.auths.EmptyAuthorizationsProvider
```

Comment:
Does the `LdapAuthorizationsProvider` need to be in here?

Reply:
no, the tutorial goes over updating this file before installing to geoserver. i think if there are two on the classpath it would throw an exception if the sys prop isn't specified.